Sources/AccioKit/Models/FrameworkProduct.swift

```diff
@@ -1,17 +1,20 @@
 import Foundation
 
 struct FrameworkProduct {
+    let framework: Framework
```

Comment:
In order to create the `.podspec` files I required to know the path to the framework (in the `Dependencies` folder) together with the framework information (dependencies, versions... etc). I found that having a reference to the `Framework` inside the `FrameworkProduct` was the easiest and safest idea, and also makes sense to me: a `FrameworkProduct` always  has an associated `Framework`.
There may be some source code improvements that can be done around now that `FrameworkProduct` contains `Framework`, but I did not include them in this PR in order to keep it as small and easy to review as possible.